doc/development/deprecations.rst

```diff
@@ -114,6 +114,12 @@ Completed deprecation cycles
   - Added in v0.32
   - Behaviour changed in v0.33
 
+* The ``prep`` keyword argument in ``QuantumScript`` has been removed.
```

Comment:
Entry needs to be removed around Line 59.

Reply:
oops, bad merge resolution. ty!